wnt.sln

```diff
@@ -39,6 +39,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "fnsock_km", "src\sock\km\fn
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "wskclient", "src\wskclient\wskclient.vcxproj", "{50F8F5EE-AA31-427F-9959-13DE0D68BD06}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "nuget", "src\nuget\nuget.vcxproj", "{ACD130FA-8DD1-461D-859E-22DFFD221921}"
```

Comment:
Nuget should not be part of the solution because it cannot do official signing between building the binaries and building the nuget. It's generally a 4 step process:

1. Build binaries.
2. Push binaries to signing system (i.e. OneBranch).
3. Build nuget with signed binaries.
4. Push nuget to signing system.

Reply:
I was following ebpf-for-windows as an example. Yeah, the above process makes more sense and the diff ended up being way cleaner :)